INSTALL.md

```diff
@@ -0,0 +1,54 @@
+# Installing the Harness CLI by downloading the binary
+
+You can install the Harness CLI (`harness`) in order to interact with Harness Platform from a command-line interface. You can install `harness` on Linux, Windows, or macOS.
+
+<!-- Regenerate this table of contents using https://github.com/ekalinin/github-markdown-toc -->
+<!-- gh-md-toc --insert README.md -->
+<!--ts-->
+
+## Table of Contents
+
+   * [Installing the CLI on Linux](#installing-the-cli-on-linux)
+   * [Installing the CLI on Windows](#installing-the-cli-on-windows)
+   * [Installing the CLI on macOS](#installing-the-cli-on-macos)
+
+
+<!-- Created by https://github.com/ekalinin/github-markdown-toc -->
+
+<!--te-->
+
+## Installing the CLI on Linux
+
+1. Navigate to [Harness CLI Releases](https://github.com/harness/harness-cli/tags) page.
+
+2. Click the release version (recommended: `latest`), your linux architecture type, and, download the archive like in the below example:
+```bash
+curl -LOJ https://github.com/harness/harness-cli/releases/download/<VERSION>/harness-<VERSION>-linux-<ARCH>.tar.gz
+```
+> NOTE: Replace `VERSION` and `ARCH` with the release version and architecture type.
+
+3. Unpack the archive:
+```bash
+tar xvzf harness-<VERSION>-linux-<ARCH>.tar.gz
+```
+
+4. Move the `harness` binary in a directory that is on your `PATH`.
+    - To check your `PATH`, execute the following command:
+        ```bash
+        echo $PATH
+        ```
+    - To move the `harness` binary
+        ```bash
+        mv ./harness /PATH/TO/DEST
```

Comment:
we are moving away from this and moving more towards PATH setup 
```
tar -xvf harness-v0.0.13-alpha-darwin-amd64.tar.gz 
export PATH="$(pwd):$PATH" 
echo 'export PATH="'$(pwd)':$PATH"' >> ~/.bash_profile
```

Reply:
cool

Reply:
The steps I provided offer users the flexibility to choose the PATH where they prefer to locate the binary. However, as we initially outlined the recommended steps in the getting started wizard, I'll mirror those steps for now. Let's aim to establish a standardized approach for installation, utilizing a setup script via curl.